tests/amqpexchange_declare_existent.phpt

```diff
@@ -23,6 +23,7 @@ try {
     $ex->setName($exchangge_name);
     $ex->setType(AMQP_EX_TYPE_TOPIC);
     $ex->declareExchange();
+    echo 'exchange ', $ex->getName(), ' declared', PHP_EOL;
```

Comment:
No, the code above should fail. This doesn't affect test case success or failure, just to make output more verbose (in case of test failure).